components/SubNav/SubNav.jsx

```diff
@@ -0,0 +1,51 @@
+require('./SubNav.scss')
+
+import React from 'react'
+import Carousel from '../Carousel/Carousel'
+import StandardListItem from '../StandardListItem/StandardListItem'
+
+const tcSubNav = {
+  compete : [
+    {img: require('./placeholder.svg'), text: 'Design Challenges', link: '/challenges/design/active'},
+    {img: require('./placeholder.svg'), text: 'Development Challenges', link: '/challenges/develop/active'},
+    {img: require('./placeholder.svg'), text: 'Data Science Challenges', link: '/challenges/data/active'},
+    // TODO dynamic domain
```

Comment:
I think right now it's just through webpack-config. We can think of a per repo solution as well, but I do kind of like having all environment vars in one spot. What do you think?